<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                           subcontrol-origin: margin;
                           padding: 0 3px;
                           subcontrol-position: top center;
                           padding-top: 0px;
                          padding-bottom: 0px;
                           padding-right: 10px;
                            color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="group_loading">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Load Focused Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="3">
         <widget class="FileFinder" name="finder_data" native="true"/>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="button_load">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="check_addToPlot">
          <property name="text">
           <string>Add to Plot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinder</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinder</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
